src/main/java/com/puppycrawl/tools/checkstyle/site/ExampleMacro.java

```diff
@@ -113,20 +111,33 @@ private static List<String> readFile(String path) throws MacroExecutionException
     }
 
     /**
-     * Extract a snippet between the given start and end delimiters.
-     * The lines containing the delimiter are not included in the snippet.
+     * Extract a configuration snippet from the given lines. Config delimiters use the whole
+     * line for themselves and have no indentation. We use equals() instead of contains()
+     * to be more strict because some examples contain those delimiters.
      *
      * @param lines the lines to extract the snippet from.
-     * @param startingDelimiter the starting delimiter.
-     * @param endingDelimiter the ending delimiter.
-     * @return the snippet.
+     * @return the configuration snippet.
      */
-    private static String getSnippetBetweenDelimiters(
-            Collection<String> lines, String startingDelimiter, String endingDelimiter) {
+    private static String getConfigSnippet(Collection<String> lines) {
         return lines.stream()
-                .dropWhile(line -> !line.contains(startingDelimiter))
+                .dropWhile(line -> !XML_CONFIG_START.equals(line))
                 .skip(1)
-                .takeWhile(line -> !line.contains(endingDelimiter))
+                .takeWhile(line -> !XML_CONFIG_END.equals(line))
```

Comment:
Explanation for changes. `contains()` is not good enough for the ending config delimeter - `*/`. There are configs that contain these two characters and the config is cutoff before the end.

For example this 
```xml
        <source>
&lt;module name="Regexp"&gt;
  &lt;property
    name="format"
    value="\A/{71}\n// checkstyle:\n// Checks Java source code for
    adherence to a set of rules\.\n// Copyright \(C\) \d\d\d\d  Oliver Burn\n
    // Last modification by \$Author.*\$\n/{71}\n\npackage [\w\.]*;\n\n
    (import [\w\.]*;\n)*\n/\*\*\n( \*[^/]*\n)* \*/"/&gt;
&lt;/module&gt;
        </source>
```

would produce the following output
```xml
&lt;module name="Regexp"&gt;
  &lt;property
    name="format"
    value="\A/{71}\n// checkstyle:\n// Checks Java source code for
    adherence to a set of rules\.\n// Copyright \(C\) \d\d\d\d  Oliver Burn\n
    // Last modification by \$Author.*\$\n/{71}\n\npackage [\w\.]*;\n\n
    (import [\w\.]*;\n)*\n/\*\*\n( \*[^/]*\n)* \*/"/&gt;
```

Reply:
Better to be specific.